tests/functional/idempotency/conftest.py

```diff
@@ -34,6 +35,18 @@ def lambda_apigw_event():
     return event
 
 
+@pytest.fixture
+def lambda_context():
+    lambda_context = {
+        "function_name": "test-func",
+        "memory_limit_in_mb": 128,
+        "invoked_function_arn": "arn:aws:lambda:eu-west-1:809313241:function:test",
```

Comment:
```suggestion
        "invoked_function_arn": "arn:aws:lambda:eu-west-1:809313241234:function:test-func",
```

Nit-picky, but the account id is 12 chars and the `function_name` is also in the ARN.

That makes me think I have the following in my project:

```python
import uuid
from aws_lambda_powertools.utilities.typing import LambdaContext


class LambdaTestContext(LambdaContext):
    def __init__(
        self,
        name: str,
        version: int = 1,
        region: str = "us-east-1",
        account_id: str = "111122223333",
    ):
        self._function_name = name
        self._function_version = str(version)
        self._memory_limit_in_mb = 128
        self._invoked_function_arn = f"arn:aws:lambda:{region}:{account_id}:function:{name}:{version}"
        self._aws_request_id = str(uuid.uuid4())
        self._log_group_name = f"/aws/lambda/{name}"
        self._log_stream_name = str(uuid.uuid4())
```
 
I use this for my tests to supply it with a valid context during testing, not sure how often we need a context but it could be useful I guess.

Reply:
@Nr18  something like this needs to be in the Powertools shared lib. It comes up a lot :)

Reply:
Yeah thats what I figured :P